### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jeznag**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const LOOK_AHEAD_MINUTES = 2, // how long before a meeting should I notify?
   PAUSE_BETWEEN_BARKS_SECONDS = 5, // how many seconds does each line of dialog have to itself
   // It can be a partial match and it's case insensitive. e.g. Holiday will match "UK holidays"
   CALENDARS_TO_EXCLUDE = ["Birthday", "Holiday", "Contacts"],
-  IS_TESTING = process.env.NODE_ENV == "test";
+  IS_TESTING = true// process.env.NODE_ENV == "test";
```
Comment (jeznag) [inline config.js:7]: probably not intended

File: index.js [hunk 2]
```diff
@@ -15,9 +15,10 @@ const {
 const quickInterval = QUICK_NAP_DURATION_SECONDS * 1000,
   slowInterval = SLOW_NAP_DURATION_MINUTES * 60_000;
 
-async function main() {
-  console.log("Late No More Online..");
+const say= require("./src/applescript/say.js")
```
Comment (jeznag) [inline index.js:18]: nitpick: space on either side of =

File: src/applescript/calendar.js [hunk 3]
```diff
@@ -1,19 +1,20 @@
-const exec = require("./exec.js"),
-  { GET_ALL_EVENTS } = require("./scripts.js");
+const exec = require('./exec.js'),
```
Comment (jeznag) [inline src/applescript/calendar.js:1]: are you using prettier to format code? I prefer single quotes too but like spaces around braces. 

CleanShot 2023-01-04 at 07 02 10

File: src/applescript/dialog.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  text,
+  buttons,
+  givingUpAfter = 30,
+  showIcon = false,
+) {
+
+  const path = files(url)
+  const SCRIPT = ` 
+--set theImage to alias of thePath as string
+set theImage to (path to temporary items as text) & "focusbear.png"
```
Comment (jeznag) [inline src/applescript/dialog.js:19]: we should be able to reference an absolute path. The swift code will put the image there. /Users/<USERNAME>/Library/Application Support/com.focusbear.latenomore/icon.png

File: src/applescript/files.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+const exec = require('./exec.js');
+
+module.exports = async function get(url) {
```
Comment (jeznag) [inline src/applescript/files.js:3]: probably don't need this

File: src/notify.js [hunk 6]
```diff
@@ -69,10 +71,12 @@ async function notifyUser(evt) {
       lastRow = i + 1 == DIALOG_STAGES.length,
       givingUpAfter = !lastRow ? perStage : 0;
 
+    console.log({line, lastRow, givingUpAfter})
     if (lastRow && !barking) startBarking(evt);
 
-    const answer = await showMeetingAlert(evt, line, givingUpAfter),
+    const answer = await showMeetingAlert(evt, line, givingUpAfter, true), // lastRow),
```
Comment (jeznag) [inline src/notify.js:77]: what's //lastRow for?

File: index.js (hunk 2 above)
Comment (jeffdechambeau) [reply index.js]: Have it fixed locally but VScode is not making this easy on me -- but it makes my eyes bleed too :)

**Pull request merged**